money-core-scala/src/main/scala/com/comcast/money/core/internal/SpanLocal.scala

```diff
@@ -20,11 +20,22 @@ import com.comcast.money.api.Span
 
 import scala.collection.mutable.Stack
 
+trait SpanLocal {
+  def current(): Option[Span]
+  def push(span: Span): Unit
+  def pop(): Option[Span]
+  def clear(): Unit
+}
+
+trait ThreadLocalSpanTracer {
+  val SpanLocal: SpanLocal = SpanThreadLocal
```

Comment:
This is really strange, having the val name collide with the trait name.  This should be renamed to something else to avoid confusion